drivers/platform/x86/thinkpad-wmi.c

```diff
@@ -319,6 +332,7 @@ static int thinkpad_wmi_simple_call(const char *guid,
 	struct acpi_buffer output = { ACPI_ALLOCATE_BUFFER, NULL };
 	acpi_status status;
 
+	status = wmi_evaluate_method(guid, 0, 0, &input, &output);
```

Comment:
why is this called twice ?

Reply:
I've added comments to the code... the is called twice to have consistent behavior for bios/acpi  between for multiple OSs.  http://www.osronline.com/showthread.cfm?link=138881 discusses the issue that triggered this change.  (essentially BIOS WMI interface was being called twice when call triggered from vbs script).

Reply:
interesting, thanks !